option.go

```diff
@@ -282,6 +302,61 @@ func (option *Option) Set(value *string) error {
 	return convert("", option.value, option.tag)
 }
 
+// Set the value of a "terminated" option to the specified value.
+// An error will be returned if the specified value could not be
+// converted to the corresponding option value type.
+// If option value is slice of slice, the parameter is converted
+// as a slice and appended to the option value. Otherwise, if
+// option value is slice, option value is replaced by parameter.
+func (option *Option) SetTerminatedOption(value []*string) error {
+	tp := option.value.Type()
+
+	if tp.Kind() != reflect.Slice {
+		return newErrorf(ErrInvalidTag,
+			"terminated flag `%s' must be a slice or slice of slices",
+			option.shortAndLongName())
+	}
+
+	if len(value) == 0 {
+		return newErrorf(ErrExpectedArgument,
+			"expected argument for flag `%s'",
+			option.shortAndLongName())
+	}
```

Comment:
Hmmm, it seems to me it that zero-length args is legitimate, kind of like `prog --foo= --bar` is valid. Someone might want to call a program with an explicit `prog --term \;` command line, and we probably shouldn't arbitrarily restrict this (with or without an explicit terminator).

That make me think: I guess terminated options can't have defaults? I guess that's fine.

Reply:
Added support for zero-length args. Thank you.

`default` can be supported quite easily, but I am wondering if it makes much sense.